README.md

```diff
@@ -192,6 +191,20 @@ class UserBlueprint < Blueprinter::Base
 end
 ```
 
+### Supporting Dynamic Blueprints for associations
+When defining an association, we can dynamically evaluate the blueprint. This comes in handy when adding polymorphic associations, by allowing reuse of existing blueprints.
+```ruby
+class TaskBlueprint < Blueprinter::Base
+  identifier :uuid
+
+  view :normal do
+    field :title, default: "N/A"
+    association :taskable, blueprint: ->(association) {association.blueprint}, default: {}
+  end
+end
+```
+Note: `association.blueprint` should return a valid Blueprint class. Currently, `has_many` is not supported because of the very nature of polymorphic associations.
+
```

Comment:
Can you also add an example on how to define `blueprint` on the associated model?